example/src/main/java/com/stripe/example/activity/PaymentActivity.java

```diff
@@ -173,4 +178,66 @@ private void initListView() {
                 new int[]{R.id.last4, R.id.tokenId});
         listView.setAdapter(simpleAdapter);
     }
+
+    private void saveCreditCard() {
+        Card cardToSave = createCardToSave();
+
+        if (validateCard(cardToSave)) {
+            startProgress();
+            new Stripe().createToken(
+                    cardToSave,
+                    PUBLISHABLE_KEY,
+                    new TokenCallback() {
+                        public void onSuccess(Token token) {
+                            addToList(token);
+                            finishProgress();
+                        }
+                        public void onError(Exception error) {
+                            handleError(error.getLocalizedMessage());
+                            finishProgress();
+                        }
+                    });
+        }
+    }
+
+    private void saveCreditCardWithRx() {
+        final Card cardToSave = createCardToSave();
+        if (!validateCard(cardToSave)) {
+            return;
+        }
+
+        final Stripe stripe = new Stripe();
+        final Observable<Token> tokenObservable =
```

Comment:
It would be nicer if I could have just returned an observable from the method in Stripe.java, but that would have forced RxJava on people rather than giving them the option. Making the Observable out of a Callable is a tiny bit of a pain, but this can be relegated to a Dagger module class.